src/constants.ts

```diff
@@ -15,8 +15,16 @@ export const BUG_REPORT_URL =
 export const FEATURE_REQUEST_URL =
 	'https://github.com/Automattic/studio/issues/new?assignees=&labels=%5BType%5D+Feature+Request&projects=&template=feature_request.yml&title=Feature+Request%3A';
 export const WPCOM_PROFILE_URL = `https://wordpress.com/me`;
+
 // OAuth constants
 export const CLIENT_ID = '95109';
 export const PROTOCOL_PREFIX = 'wpcom-local-dev';
 export const WP_AUTHORIZE_ENDPOINT = 'https://public-api.wordpress.com/oauth2/authorize';
 export const SCOPES = 'global';
+
+// Time-related constants
+export const HOUR_MS = 1000 * 60 * 60;
+export const DAY_MS = HOUR_MS * 24;
+
+// AI Assistant constants
+export const CLEAR_HISTORY_REMINDER_TIME = 2 * HOUR_MS; // In milliseconds
```

Comment:
We have the same `HOUR_MS` in `use-expiration-date.ts` file. Let's reuse those variables there, so we don't have the same variables defined twice.

Reply:
Scratch that. I scrolled :smile:

Reply:
Yep, I updated `use-expiration-date.ts` after introducing the time-related constants.